config/prod/gene_drug_db_test_instance.yaml

```diff
@@ -0,0 +1,20 @@
+# Provision an EC2 instance
+template_path: templates/managed-ec2-v2.yaml
+stack_name: gene_drug_db_test_instance
```

Comment:
A stack name can contain only alphanumeric characters (case-sensitive) and hyphens.  Also would you mind adding `-ec2` to the end of the stack name and to the corresponding filename?

Reply:
👍 This also should be fixed now.